components/script/dom/readablestreamdefaultreader.rs

```diff
@@ -176,16 +197,19 @@ impl ReadableStreamDefaultReader {
 
     /// <https://streams.spec.whatwg.org/#readable-stream-add-read-request>
     pub fn add_read_request(&self, read_request: ReadRequest) {
+        error!("ReadableStreamDefaultReader::add_read_request");
```

Comment:
This is confusing because I don't see any places in the code where read_rrquests has a borrow that lasts outside of a single statement. I can't figure out where there would be a conflicting borrow in this case.

Reply:
Since the panic is coming from self.read_requests.borrow_mut(), it's only other calls that use self.read_requests that matter.

Reply:
Can you paste the full backtrace from the panic? The conflicting borrow will be in one of those stack frames.